zerver/lib/actions.py

```diff
@@ -805,10 +805,13 @@ def do_send_messages(messages_maybe_none):
             # mentioned; we will want to change that structure.
             for user_profile in message['active_recipients']:
                 trigger = None
+                sender = message['message'].sender
                 if not user_profile.is_bot:
                     continue
                 if user_profile.bot_type != UserProfile.OUTGOING_WEBHOOK_BOT:
                     continue
+                if sender.is_bot:
+                    continue
```

Comment:
These three lines look unrelated to this pull request.  Can you please remove them from this PR and put this logic in a separate PR?  (Also, in the separate PR, I think the logic can go outside of the `for` loop.)

One possible name for the separate PR could be "Prevent outgoing webhooks from triggering if sender is a bot."

By extracting this logic into a separate PR, there are two advantages:
(1) It's more clear what each individual PR does
(2) We will be able to merge each individual PR much more quickly

Thanks!